compiler-rt/lib/builtins/divsc3.c

```diff
@@ -20,7 +20,7 @@
 COMPILER_RT_ABI Fcomplex __divsc3(float __a, float __b, float __c, float __d) {
   int __ilogbw = 0;
   float __logbw =
-      __compiler_rt_logbf(__compiler_rt_fmaxf(crt_fabsf(__c), crt_fabsf(__d)));
+      __compiler_rt_logbf(__compiler_rt_fmaxX(crt_fabsf(__c), crt_fabsf(__d)));
```

Comment:
See also https://reviews.llvm.org/D91841, which added the relevant code.

If you're going to delete the last use of __compiler_rt_fmaxf, please also delete the function itself.

Reply:
Done